### DISCUSSION TITLE
Welcome to GEL-next Discussions!

### DISCUSSION THREAD

**Discussion by samithaf** [WestpacGEL/GEL-next#162] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by vbapure** [comment]
Hi,
 My name is Vinod Bapure.
 I am excited to be part of this community.
 Looking to hear and share ideas. Contribute in whatever little way possible.

Thanks

**Reply by samithaf** [reply]
@vbapure Thanks! Please keep eye on `Good first issue` tagged issues.

**Comment by andy008** [comment]
Hi,
I'm working with Ive Group, a vendor providing web forms replacing PDFs. For example https://psr.westpacgroup.com.au
We have an extensive Angular code base across multiple from SPA web apps using the previous GEL implementation (not React).
Ideally, we'd like Angular components, or a vanilla HTML/CSS library... or we may just adapt the existing React components be rebuilding them in Angular.
Is there anything in progress with the non-React implementation of the GEL originally promised?
Thanks!

**Reply by samithaf** [reply]
Hello @andy008 Westpac GEL based on [React.js](https://react.dev/). We don't any intention to create vanilla HTML/CSS or to support any other JS library/framework other than React.js.
